### DISCUSSION TITLE
Hi ! Can someone pls help me for Kade Engine ?

### DISCUSSION THREAD

**Discussion by BasmaRuto** [FunkinCrew/Funkin#1868] [Compiling help]
So i used the command "lime test windows" and it seem worked.
I waited until the end and, got to the end, it didn't work and it put me this :

Error: serialization.cpp C:\HaxeToolkit\haxe\lib\discord_rpc\git\lib\discord-rpc\src\serialization.h(13): fatal error C1083: Impossible d'ouvrir le fichier include : 'rapidjson/document.h' : No such file or directory   - C:/HaxeToolkit/haxe/lib/discord_rpc/git/lib/discord-rpc/src/rpc_connection.cpp Error: discord-rpc.cpp C:\HaxeToolkit\haxe\lib\discord_rpc\git\lib\discord-rpc\src\serialization.h(13): fatal error C1083: Impossible d'ouvrir le fichier include : 'rapidjson/document.h' : No such file or directory

And i don't know why, i swear when I manage to solve a problem, another one appears...
Any help would be appreciated !

**Comment by isakube** [comment]
Wrong repo. KadeDev/Kade-Engine

**Reply by BasmaRuto** [reply]
idk how to delete this post

**Reply by libauror-a** [reply]
you cannot delete posts. your trace is never erased from existance

**Reply by Jotaro-Gaming** [reply]
> you cannot delete posts. your trace is never erased from existance

nm99 **are ya sure bout dat?**
